src/language-js/print/jsx.js

```diff
@@ -177,16 +177,16 @@ function printJsxElementInternal(path, options, print) {
   // Trim trailing lines (or empty strings)
   while (
     children.length > 0 &&
-    (isLineNext(getLast(children)) || isEmpty(getLast(children)))
+    (!getLast(children) || isLineNext(getLast(children)))
   ) {
     children.pop();
   }
 
   // Trim leading lines (or empty strings)
   while (
     children.length > 0 &&
-    (isLineNext(children[0]) || isEmpty(children[0])) &&
-    (isLineNext(children[1]) || isEmpty(children[1]))
+    (!children[0] || isLineNext(children[0])) &&
```

Comment:
Just a question, is `!children[0]` enough instead of `children[0] !== ""` ?

Reply:
`children` is a `Doc` here. Only false value is `""`